gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/management/JdbcThemeRepository.java

```diff
@@ -47,6 +48,7 @@ protected JdbcObjectMapper<Theme> buildOrm() {
             .builder(Theme.class, this.tableName, "id")
             .addColumn("id", Types.NVARCHAR, String.class)
             .addColumn("name", Types.NVARCHAR, String.class)
+            .addColumn("type", Types.NVARCHAR, ThemeType.class)
```

Comment:
Check that 'type' is not a protected field in some DB. Look for another column named 'type' and look at the way it has been declared and handled

Reply:
We have a column`"type"` for the `JdbcApiRepository` and no special configuration/handling. So I figure it should be ok .